client/reader/follow-button/index.jsx

```diff
@@ -16,9 +16,8 @@ const ReaderFollowButton = React.createClass( {
 	mixins: [ PureRenderMixin ],
 
 	propTypes: {
-		following: React.PropTypes.bool.isRequired,
```

Comment:
I think it's more confusing, as it's only applicable when using the button, not the container. The underlying component will yell if you pass the wrong thing.